Source/WebCore/platform/graphics/FontCascadeFonts.h

```diff
@@ -100,8 +101,8 @@ class FontCascadeFonts : public RefCounted<FontCascadeFonts> {
         std::unique_ptr<MixedFontGlyphPage> m_mixedFont;
     };
 
-    GlyphPageCacheEntry m_cachedPageZero;
-    HashMap<int, GlyphPageCacheEntry> m_cachedPages;
+    EnumeratedArray<ResolvedEmojiPolicy, GlyphPageCacheEntry, ResolvedEmojiPolicy::RequireEmoji> m_cachedPageZero;
+    EnumeratedArray<ResolvedEmojiPolicy, HashMap<int, GlyphPageCacheEntry>, ResolvedEmojiPolicy::RequireEmoji> m_cachedPages;
```

Comment:
We should hook `EnumeratedArray` into `WTF::EnumTraits` so it can infer the max value

Reply:
https://github.com/WebKit/WebKit/pull/15809